config/application.rb

```diff
@@ -68,6 +70,11 @@ class Application < Rails::Application
     config.middleware.use PDFKit::Middleware, { print_media_type: true, page_size: "A4" }, disposition: 'attachment', only: [%r[^/provider/applications/\d+]]
     config.skylight.environments = ENV['SKYLIGHT_ENABLE'].to_s == 'true' ? [Rails.env] : []
 
+    config.action_dispatch.trusted_proxies = [
```

Comment:
Can you move this to only the prod environment: https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/config/environments/production.rb#L88

Reply:
example: https://github.com/DFE-Digital/teaching-vacancies/blob/master/config/environments/production.rb#L94

Reply:
I've shifted it over